### DISCUSSION TITLE
Add LCOM metric

### DISCUSSION THREAD

**Discussion by DeGraciaMathieu** [Halleck45/ast-metrics#31] [Ideas]
Hi,

When I analyze my projects, I often start by examining the application's structure using LCOM.

Is there any plan to add this metric to the tool ?

**Comment by Halleck45** [comment]
Yes, I think it's a "must have". I've added external coupling for PHP. Adding LCOM seems possible, with a bit of work.

**Comment by Halleck45** [comment]
Hi ! 

This is implemented in https://github.com/Halleck45/ast-metrics/pull/94
